core/templates/dev/head/pages/dashboard/Dashboard.js

```diff
@@ -84,14 +84,8 @@ oppia.controller('Dashboard', [
       $scope.activeTab = newActiveTabName;
     };
 
-    $scope.showExplorationEditor = function(explorationId) {
-      $window.location = '/create/' + explorationId;
-    };
-
-    $scope.openExploration = function(status, explorationId) {
-      if (status === 'private') {
-        $scope.showExplorationEditor(explorationId);
-      } else {
+    $scope.getStatsForPublishedExp = function(status, explorationId) {
```

Comment:
Suggest: Rename to `getStatsForNonPrivateExp` to better explain what this function is doing!